Agents/ForecastingAgent/tests/test_integration.py

```diff
@@ -517,3 +517,174 @@ def test_create_tft_forecast(
         print(f'{k}: {round(v,5)}')
 
     print("All check passed.")
+
+
+#pytest.mark.skip(reason="")
+@pytest.mark.parametrize(
+    "derivation_input_set, dataIRI, input_chunk_length, with_unit, overwrite_forecast, ts_times, covariates, case",
+    [
+        (cf.DERIVATION_INPUTS_7, cf.IRI_TO_FORECAST_1, cf.DURATION_2, False, True, cf.TIMES, cf.COVARIATES_2, cf.TEST_CASE_13),
```

Comment:
Are we currently testing how forecasting with covariates compares to without? Although this is highly use case specific (i.e., depends on correlation of covariates), I think this would be a useful test to see how/that covariates actually improve performance